.env.copy

```
@@ -1,51 +1,30 @@
-
-# This is the single source of truth to know where the tf we at rn, NODE_ENV get shiesty
 NEXT_PUBLIC_CURRENT_ENV=
-
+NEXT_PUBLIC_WWW_URL=
+NEXT_PUBLIC_BLOG_URL=
+NEXT_PUBLIC_WWW_GOOGLE_ANALYTICS_ID=
+NEXT_PUBLIC_BLOG_GOOGLE_ANALYTICS_ID=
+NEXT_PUBLIC_SENTRY_DSN=
+NEXT_PUBLIC_POSTHOG_KEY=
+NEXT_PUBLIC_POSTHOG_HOST=
+NEXT_PUBLIC_LOGTAIL_INGESTION_TOKEN=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_LOGTAIL_INGESTION_TOKEN key should go before the NEXT_PUBLIC_POSTHOG_HOST key

(UnorderedKey)